API.md

```diff
@@ -0,0 +1,337 @@
+# Microsoft.IO.RecyclableMemoryStream.dll v.1.3.2.0 API documentation
+
+Created by 
+[mddox](https://github.com/loxsmoke/mddox) on 2/3/2020
+
+# All types
```

Comment:
move the types that have only auto generated docs to the bottom.  If that will cause an issue with future generation refreshes, consider a quick index here of the most interesting types (which are the only ones people will actually look at)

Reply:
Yeah...I've been trying to find a good XML->MD translator and they mostly leave a lot to be desired. Trying a new one now that should resolve this.